local-cli/generator/index.js

```diff
@@ -6,9 +6,13 @@ var yeoman = require('yeoman-generator');
 module.exports = yeoman.generators.NamedBase.extend({
   constructor: function() {
     yeoman.generators.NamedBase.apply(this, arguments);
+    this.option('swift');
+  },
 
+  composing: function() {
+    var generator = this.options.swift ? 'generator-swift' : 'generator-ios';
```

Comment:
they're both ios.
it should be something like `'generator-ios-swift' : 'generator-ios-objc'`